### ISSUE TITLE
[hw3] testing에서 src/test-utils/*을  빼도 되는지 여쭙고 싶습니다.

### ISSUE THREAD

**Issue by pacifiic** [swsnu/swppfall2022#149]
practice에서 package.jason에 한 데로 
"jest": {
    "collectCoverageFrom": [
      "src/**/*.{js,jsx,ts,tsx}",
      "!src/index.tsx",
      "!src/test-utils/*"
    ]
  }
를 추가하고 test를 했습니다.
practice와 같이 test-utils를 제하고 test해도 되는지 여쭙고 싶습니다.

**Comment by gajagajago** [comment]
cc. @ccdc-git

**Comment by ccdc-git** [comment]
네 상관없습니다. 다만 test-utils의 test coverage가 100%이 아니라면 test-utils의 코드를 충분히 활용한게 아니므로 utils를 제대로 사용하고 있는지 확인해주세요.

**Comment by pacifiic** [comment]
> 네 상관없습니다. 다만 test-utils의 test coverage가 100%이 아니라면 test-utils의 코드를 충분히 활용한게 아니므로 utils를 제대로 사용하고 있는지 확인해주세요.

"다만 test-utils의 test coverage가 100%이 아니라면 test-utils의 코드를 충분히 활용한게 아니므로 utils를 제대로 사용하고 있는지 확인해주세요." -> 죄송하지만 무슨 의미인지 잘 이해를 못 했습니다. 모든 파일에서 test-utils를 호출해야 한다는 의미입니까?
